src/net.cpp

```diff
@@ -1822,7 +1826,7 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 LogPrint(BCLog::NET, "Making feeler connection to %s\n", addrConnect.ToString());
             }
 
-            OpenNetworkConnection(addrConnect, (int)setConnected.size() >= std::min(nMaxConnections - 1, 2), &grant, nullptr, false, fFeeler);
```

Comment:
My intuition is that we have bigger problems than not receiving transactions if we're trying to connect to 8 peers but can only make 2 connections, so this isn't a big deal -- happy to reconsider if others disagree.

Reply:
I mean I dont think its all that critical, but I do think I'd prefer if it were non-blocks-only first. Specifically, if you have a rather-stale addrman, you'd like to use your first few peers to exchange addresses so speed up getting further connections (instead of being stuck spinning).

Reply:
Fixed in the latest version.